docs/docs-components/clipboardCopy.js

```diff
@@ -52,5 +52,6 @@ export default function clipboardCopy(text: string): Promise<void> {
 
   return success
     ? Promise.resolve()
-    : Promise.reject(new DOMException('The request is not allowed', 'NotAllowedError'));
+    : // $FlowIssue[cannot-resolve-name] DOMException is not part of the internal flow definitions
```

Comment:
HERE

Reply:
looks like widely supported https://developer.mozilla.org/en-US/docs/Web/API/DOMException/DOMException#browser_compatibility but not defined in https://github.com/facebook/flow/blob/main/lib/dom.js

`$FlowIssue` is accurate since this is a Flow internal issue.

LGTM